tests/devices/test_null_qubit.py

```diff
@@ -129,6 +117,28 @@ def small_circ(params):
         }
     )
 
+    captured = capsys.readouterr()
+    captured = captured.out.splitlines()
+
+    RESOURCE_PRINT_DELIMETER = "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
```

Comment:
Could you use RESOURCE_PRINT_DELIMETER of the null.qubit device instead?

Reply:
Fixed in [7c6bc8d](https://github.com/PennyLaneAI/pennylane/pull/7392/commits/7c6bc8df8f29919856cc8c4ae89a6cde660b8e5a)